.github/workflows/codeql-analysis.yml

```diff
@@ -11,6 +11,11 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
+        
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17.2
```

Comment:
Perhaps set it to just 1.17?

For reference, this is what we have in the core repository:

https://github.com/open-telemetry/opentelemetry-collector/blob/c24dfeb047a8d0ca4c5a210c305be8ea80c10994/.github/workflows/codeql-analysis.yml#L15-L18

Reply:
No particular reason to pin the bugfix version, I updated it to be the same as in core